libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Views/SFSDKWindowManager.m

```diff
@@ -32,11 +32,81 @@
 #import "SFApplicationHelper.h"
 #import "SFSecurityLockout.h"
 
+@interface SFSDKUIWindow ()
+- (instancetype)initWithFrame:(CGRect)frame;
+- (instancetype)initWithFrame:(CGRect)frame andName:(NSString *)windowName;
+- (void)stashRootViewController;
+- (void)unstashRootViewController;
+
+@property (nonatomic,strong) UIViewController * _Nullable stashedController;
+@property (nonatomic,readonly) NSString * _Nullable windowName;
+@end
+
+@implementation SFSDKUIWindow
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        _windowName = @"NONAME";
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame andName:(NSString *)windowName {
+    self = [super initWithFrame:frame];
+    if (self) {
+        _windowName = windowName;
+    }
+    return self;
+}
+
+- (void)stashRootViewController {
```

Comment:
Could you put the comment from the PR in this class to explain what is going on and why we are doing it this way?

Reply:
This stash function is called when the window is not the key responder anymore. When another window is make key & visible, UIWindow:resignKeyWindow will be invoked by UIKIT. This is when we stash(copy then nullify the reference) for the rootviewcontroller.

Reply:
What I meant earlier, is could you add the comments in the source code so that our future selfs can make sense of the code without having to do a git blame etc.